### DISCUSSION TITLE
Missing Total Commits depending on hide_border[I can't comment...]

### DISCUSSION THREAD

**Discussion by APN-Pucky** [anuraghazra/github-readme-stats#1968] [Q&A] [answered]
```
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=APN-Pucky&show_icons=true&include_all_commits=true&theme=buefy&hide_border=true)](https://github.com/anuraghazra/github-readme-stats)
```
produces:

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=APN-Pucky&show_icons=true&include_all_commits=true&theme=buefy&hide_border=true)](https://github.com/anuraghazra/github-readme-stats)

while

```
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=APN-Pucky&show_icons=true&include_all_commits=true&theme=buefy&hide_border=tru)](https://github.com/anuraghazra/github-readme-stats)
```
produces

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=APN-Pucky&show_icons=true&include_all_commits=true&theme=buefy&hide_border=tru)](https://github.com/anuraghazra/github-readme-stats)

The only difference is that the `hide_border` parameter needs to be misspelled for Total Commits to show up.

Image in case the live version changes:
![2022-08-22 21_01_08-New](https://user-images.githubusercontent.com/4533248/185998456-df8df7b4-91c9-4a27-820a-e9c9f9036725.png)

EDIT: I just realized that this might just be due to some caching from Github.

**Comment by rickstaa** [comment, marked as answer]
@APN-Pucky Thanks for your question. I checked your syntax and everything seems fine because you use `hide_border=tru` instead of `hide_border=true`. Everything seems fine. I, therefore, don't thithe `hide_border` flag causes itlag. The most likely candidate for causing the commits to display as `0` is the `include_all_commits` flag. If this flag is enabled, we use the `totalCommitsFetcher` to get all the commits. 

https://github.com/anuraghazra/github-readme-stats/blob/e9c55d90ad8c9663a480ffaa56484f979ce86fad/src/fetchers/stats-fetcher.js#L68-L98

This function uses the [Github Rest API](https://docs.github.com/en/rest) instead of [GitHub's GraphQl API](https://docs.github.com/en/graphql). If this API throws an error, we return 0.

https://github.com/anuraghazra/github-readme-stats/blob/e9c55d90ad8c9663a480ffaa56484f979ce86fad/src/fetchers/stats-fetcher.js#L93-L96

The best thing to do when you see this issue happen is to open the console and check which error is thrown. I hope that solves the problems you were having. Feel free to post the error below if the issue happens again.

**Reply by rickstaa** [reply]
> ### Discussed in #1968
> @rickstaa I edited my issue/discussion, but it is locked so I can't comment, but only edit my texts: I realized that this issues might just be due to some caching from Github.
> 
> This issue can be deleted/closed, but I wanted to notify you about this for future issues->discussion.
